src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -164,12 +164,6 @@ public BuildScriptSnippet GenerateBashBuildScriptSnippet(BuildScriptGeneratorCon
                 configureYarnCache = true;
                 packageInstallerVersionCommand = NodeConstants.YarnVersionCommand;
             }
-            else if (StaticSiteGeneratorHelper.IsHugoApp(ctx.SourceRepo, _environment))
-            {
-                packageManagerCmd = NodeConstants.HugoCommand;
-                packageInstallCommand = NodeConstants.HugoCommand;
-                packageInstallerVersionCommand = NodeConstants.HugoVersionCommand;
```

Comment:
we could remove these constants from NodeConstants if they're not used anymore. Ignore if they're already removed.

Reply:
Sure, I will handle this clean up in a separate PR.